Dockerfile

```diff
@@ -43,12 +43,12 @@ RUN go build  \
 
 
 FROM scratch AS elemental-operator
-COPY --from=build /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/
+COPY --from=build /var/lib/ca-certificates/ca-bundle.pem /etc/ssl/certs/ca-certificates.crt
```

Comment:
The bundled certificate file is not at the same place in SUSE based image. I keep it in `/etc/ssl/certs/ca-certificates.crt` as before to avoid issues, but is it a good idea?

Reply:
I think it should not matter, its on the search path, isnt it? I believe golang just uses all the certs under that path to search for validity, so it should be good to go.

Reply:
Ok, so I leave it as is, it should work as expected.